ws-nextjs-app/pages/[service]/send/[id]/FormField/index.tsx

```diff
@@ -35,6 +39,7 @@ const FormComponents: Record<
   checkbox: Checkbox,
   phone: Telephone,
   textarea: TextArea,
+  file: File,
```

Comment:
I don't think this will match as we're matching on `htmlType` below, which the `file` schema doesn't include.

Reply:
We can maybe leave as is for now and follow up a fix. Think we probably should work out a way on the back-end to return the `htmlTypes` for these unique input fields.